src/dataExplorer/components/Results.tsx

```diff
@@ -200,14 +228,21 @@ const WrappedOptions: FC = () => {
 }
 
 const GraphHeader: FC = () => {
-  const {view, setView} = useContext(ResultsViewContext)
+  const {view, setView, viewOptions} = useContext(ResultsViewContext)
   const {result} = useContext(ResultsContext)
   const {result: subQueryResult} = useContext(ChildResultsContext)
   const {launch} = useContext(SidebarContext)
 
+  const dataExists = !!result?.parsed
+
   const launcher = () => {
     launch(<WrappedOptions />)
   }
+  useEffect(() => {
+    if (dataExists) {
+      launcher()
+    }
+  }, [viewOptions])
```

Comment:
This causes the sidebar (graph customize) to autolaunch when the graph view is open.